common/mariadb/templates/pre-change-job.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if or .Values.databases .Values.users }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ include "fullName" . }}-pre-change"
```

Comment:
Don't we need some special naming (a variable in the name e.g. chart version), because we're unable to update Jobs with helm3?

Reply:
No, the annotation `"helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded` takes care of that.
The job will be deleted when:
- the hook will be executed (so before the upgrade), in case it exists
- the hook has been successfully executed

In other words, helm should only leave the job around, if the hook didn't succeed, so one can debug it.
So one can simply retry the execution without having to manually delete the job, I've added `before-hook-creation` .

(Something which I think should also be added to https://github.com/sapcc/helm-charts/blob/master/common/region_check/templates/region-check-job.yaml#L16)